chart/compass/charts/kyma-environment-broker/values.yaml

```
@@ -48,8 +48,10 @@ additionalRuntimeComponents:
     namespace: "kyma-system"
   - name: "compass-runtime-agent"
     namespace: "compass-system"
+  - name: "knative-eventing-kafka"
+    namespace: "knative-eventing"
 
-kymaVersion: "1.10.0"
+kymaVersion: "master-99c5dca2"
```

```suggestion
kymaVersion: "1.11.0-rc2"
```

I've tested that with rc2 and more or less it's working